frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDasboard.tsx

```diff
@@ -29,16 +31,19 @@ const TeamEventCreditDashboard = (props: {
     remainingCredits = REQUIRED_COMMUNITY_CREDITS - approvedCommunityCredits;
   else remainingCredits = 0;
 
+  let headerString;
+  if (userRole !== 'lead')
+    headerString = `Check your team event credit status for this semester here! Every DTI member must complete ${REQUIRED_TEC_CREDITS} team event credits and ${REQUIRED_COMMUNITY_CREDITS} community team event credits to fulfill this requirement.`;
+  else
+    headerString =
+      'Since you are a lead, you must complete 6 total team event credits, with 1 of them being community event credits.';
```

Comment:
Not 100% about this but I thought they got rid of the community event requirement. Otherwise +1 on @andrew032011 comment. The "6" in the `headerString` for leads can just be replaced with `REQUIRED_TEC_CREDITS`

Reply:
They are postponing the community event requirement this year. I'll hurry up and get a PR out to address that so there's no more confusion.

Reply:
I replaced them with `REQUIRED_COMMUNITY_CREDITS` for now.